plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudgetingDefaults.hs

```diff
@@ -90,7 +91,16 @@ unitCekParameters =
     noinline mkMachineParameters def $
         CostModel unitCekMachineCosts unitCostBuiltinCostModel
 
-defaultBuiltinsRuntime :: HasMeaningIn DefaultUni term => BuiltinsRuntime DefaultFun term
+defaultBuiltinsRuntimeForSemanticsVariant
```

Comment:
@effectfully Do we maybe want to make this the thing that's actually exported and get the user to say which variant to use?  `BuiltinSemanticsVariant2` is what gets used by default at the moment.